acceptance/helpers/delete_clusters/delete_clusters.go

```diff
@@ -0,0 +1,227 @@
+// Copyright © 2021 - 2023 SUSE LLC
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/epinio/epinio/acceptance/helpers/proc"
+	"github.com/epinio/epinio/acceptance/helpers/route53"
+)
+
+func main() {
+	runID, runIDcheck := os.LookupEnv("RUN_ID")
+	pcp, pcpcheck := os.LookupEnv("RUN_PCP")
+	if !runIDcheck || !pcpcheck {
+		if !runIDcheck {
+			fmt.Println("Error: Variable RUN_ID not set in env")
+		}
+		if !pcpcheck {
+			fmt.Println("Error: Variable RUN_PCP not set in env")
+		}
+		return
+	}
+
+	DeleteCluster(runID, pcp)
+}
+
+func CleanupDNS(zoneID string, domainname string) {
+	dnsrecords := [2]string{domainname + ".", "\\052." + domainname + "."}
+	fmt.Println("Cleaning up AWS Route53 DNS records ...")
+
+	for _, dnsrecord := range dnsrecords {
+		Name, Type, Record, err := route53.GetRecord(zoneID, dnsrecord)
+		if err != nil {
+			fmt.Println("Error: ", err)
+		}
+		if Name != "Clean" {
+			nodeTmpDir := ""
+			switch Type {
+			case "A":
+				change := route53.A(Name, Record, "DELETE")
+				out, err := route53.Update(zoneID, change, nodeTmpDir)
+				if err != nil {
+					fmt.Println("Error: ", err, out)
+				} else {
+					fmt.Println("Cleaned up AWS Route53 DNS record: ", dnsrecord)
+				}
+			case "CNAME":
+				change := route53.CNAME(Name, Record, "DELETE")
+				out, err := route53.Update(zoneID, change, nodeTmpDir)
+				if err != nil {
+					fmt.Println("Error: ", err, out)
+				} else {
+					fmt.Println("Cleaned up AWS Route53 DNS record: ", dnsrecord)
+				}
+			}
```

Comment:
I would rewrite like this, since it seems that the logs/error are the same, and that the nodeTmpDir is not used:
```suggestion
			switch Type {
			case "A":
				change := route53.A(Name, Record, "DELETE")
			case "CNAME":
				change := route53.CNAME(Name, Record, "DELETE")
			}
			
			out, err := route53.Update(zoneID, change, "")
			if err != nil {
				fmt.Println("Error: ", err, out)
			} else {
				fmt.Println("Cleaned up AWS Route53 DNS record: ", dnsrecord)
			}
```